kafka-connect/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fedora:25
+
+RUN dnf -y install which java-1.8.0-openjdk libaio python gettext hostname iputils wget && dnf clean all -y
+
+# set Scala and Kafka version
+ENV SCALA_VERSION=2.11
+ENV KAFKA_VERSION=0.11.0.1
+# set Kafka home folder
+ENV KAFKA_HOME=/opt/kafka
+
+# Set from build args
+ARG version=latest
+ENV VERSION ${version}
+
+# downloading/extracting Apache Kafka
+RUN wget http://www.eu.apache.org/dist/kafka/$KAFKA_VERSION/kafka_$SCALA_VERSION-$KAFKA_VERSION.tgz
+
+RUN mkdir $KAFKA_HOME \
+    && tar xvfz kafka_$SCALA_VERSION-$KAFKA_VERSION.tgz -C $KAFKA_HOME --strip-components=1 \
+    && mkdir $KAFKA_HOME/plugins
+
```

Comment:
Shouldn't we `RUN useradd` and `USER` to run connect as non-root?

Reply:
I'm not sure how is the Fedora base image built. It seems to currently run under some user `1000050000` in root group. But doesn't look to have all root permissions. I think we should look into it together with the base image, since this can be done in the base image

Reply:
I agree, lets do that at the same time as the base image.